packages/@uppy/react/src/DashboardModal.js

```diff
@@ -31,6 +31,12 @@ class DashboardModal extends React.Component {
     } else if (!prevProps.open && this.props.open) {
       this.plugin.openModal()
     }
+
+    for (const key of Object.keys(this.props)) {
+      if (this.props[key] !== prevProps[key] && key in this.plugin.opts) {
+        this.plugin[key] = this.props[key]
```

Comment:
Are we sure we want to do this for everything? This may have unexpected consequences in the future. Perhaps it's better to just explicitly update the value in question.

Reply:
We pass all the props to the dashboard plugin options, so that's why I thought it would make sense to do that. We could also try to come up with a list of options that we think users may want/need to mutate (but that's harder to maintain), or uninstall+re-install the plugin for any props change (but that's probably not very efficient).